pkg/v1/tkg/client/upgrade_region.go

```diff
@@ -338,7 +338,7 @@ func (c *TkgClient) WaitForAddonsDeployments(clusterClient clusterclient.Client)
 }
 
 // WaitForPackages wait for packages to be up and running
-func (c *TkgClient) WaitForPackages(regionalClusterClient, currentClusterClient clusterclient.Client, clusterName, namespace string) error {
+func (c *TkgClient) WaitForPackages(regionalClusterClient, currentClusterClient clusterclient.Client, clusterName, namespace string, isRegionalCluster bool) error {
```

Comment:
I hope we're not breaking external clients with this change.

Reply:
`client` is not getting consumed directly from external clients but the `tkgctl` package gets used by external clients. So, we should be good here.